src/annotator/highlighter.js

```diff
@@ -210,19 +237,19 @@ export function highlightRange(normedRange, cssClass = 'hypothesis-highlight') {
  *
  * nb. This is like `ChildNode.replaceWith` but it works in IE 11.
  *
- * @param {Node} node
+ * @param {ChildNode} node
  * @param {Node[]} replacements
  */
 function replaceWith(node, replacements) {
-  const parent = node.parentNode;
+  const parent = /** @type {Node} */ (node.parentNode);
```

Comment:
does this need a null check or is a cast okay?

Reply:
In this case I believe it is safe to assume the node has a parent.